src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -557,6 +557,23 @@ describe('manage nsfs cli account flow', () => {
             expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgumentType.message);
         });
 
+        it('cli unset new_buckets_path', async () => {
```

Comment:
Minor suggestion so it will be easier to understand the test when only viewing the test's name.
```suggestion
        it('cli update account unset new_buckets_path', async () => {
```